package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>libopencv-dev</name>
+  <version>4.2.0</version>
+  <description>
+    Open Source Computer Vision Library.
+  </description>
+  <maintainer email="opencv@opencv.org">opencv</maintainer>
+  <license>BSD</license>
+
+  <buildtool_depend>cmake</buildtool_depend>
+  
+  <build_depend>eigen</build_depend>
```

Comment:
Is this list really complete?

Reply:
As @mikaelarguedas https://github.com/opencv/opencv/pull/16322#issuecomment-575364146 pointed out, we could specify more of the optional dependencies, I just wasn't sure to what extent  we'd like to include them explicitly as required, though the one linked seems like a more practical useful build for general users.